include/RAJA/RAJA.hpp

```diff
@@ -26,6 +26,8 @@
 #ifndef RAJA_HPP
 #define RAJA_HPP
 
+
+
```

Comment:
I'm being nitpicky here, but can we remove the extra blank lines introduced across the files?

Reply:
sure, i can take a pass at that